
camera_frame: 'camera_link'
object_frame: 'object_link'
camera_topic: 'puzzlebot/camera/image_raw'
target_id: 4

desired_corner_locations: [[381., 435.], 
                            [243., 435.], 
                            [240., 301.], 
                            [384., 302.]]

z_desired: 0.18

focal_length_pixel: 530.466941
camera_matrix: [[530.466941, 0, 205], 
                [0, 530.466941, 154], 
                [0, 0, 1]]

distortion_coeffs: [0.0, 0.0, 0.0, 0.0, 0.0]

visual_controller_topic: '/corners'
control_rate: 30

visual_servoing:
  # Proportional controller
  kp: 0.6

  # Tolerances
  e_tolerance: 1
  
  commands_topic: '/puzzlebot/cmd_vel'

# kmodel_rate: 50
# odom_rate: 50
# rviz_rate: 50
# wheel_radius: 0.05
# track_length: 0.19
# inertial_frame_name: 'odom'
# robot_frame_name: 'base_link'
# commands_topic: 'cmd_vel'
# pose_topic: 'pose'
# wl_topic: 'wl'
# wr_topic: 'wr'
# odometry_topic: 'odom'

# joint_state_pub:
#   joint_states_topic: '/joint_states'
#   joint_names: ['leftWheel', 'rightWheel']
#   joint_initial_positions: [0.0, 0.0]
